### ISSUE TITLE
I broke something

### ISSUE THREAD

**Issue by matthazinski** [andreimarcu/linx-client#6]
```
[matt@teselecta ~]$ ./linx-client-v1.1.2_linux-amd64 
Configuring linx-client

Site url (ex: https://linx.example.com/): https://linx.vtluug.org/
Logfile path (ex: ~/.linxlog): 
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x403b4b]

goroutine 1 [running]:
main.parseConfig()
	/Users/andreimarcu/localdev/go/src/github.com/andreimarcu/linx-client/linx.go:218 +0xecb
main.main()
	/Users/andreimarcu/localdev/go/src/github.com/andreimarcu/linx-client/linx.go:44 +0x24
```

This happens everytime I try to run on Arch amd64 regardless of whether ~/.logfile exists and whether I specify ~/.logfile or leave it blank.

**Comment by andreimarcu** [comment]
Found it, my binary releases are affected by https://code.google.com/p/go/issues/detail?id=6376
I'll remove them for now and either do the current user differently (or not at all) or fix the build system.

**Comment by andreimarcu** [comment]
Fixed in 2b984a26fa89f5c0d1c1e4ee1bb7b3ff598964a3
